iocore/net/UnixNetAccept.cc

```diff
@@ -251,13 +251,23 @@ NetAccept::init_accept_per_thread()
   }
 }
 
+//
+// Stop NetAccept and its slave objects
+//
 void
 NetAccept::stop_accept()
 {
-  if (!action_->cancelled) {
-    action_->cancel();
+  forl_LL(NetAccept, na, slaves)
+  {
+    na->ep.stop();
   }
-  server.close();
+  ep.stop();
```

Comment:
We should call `ep.stop()` before `server.close()`, or the `server.fd` will not delete from epoll fd because the server.fd did not really closed. The `server.fd` is shared with `traffic_manager` processor.